### DISCUSSION TITLE
Uppercase and Lowercase

### DISCUSSION THREAD

**Discussion by faceincase** [ORelio/Minecraft-Console-Client#1801] [Q&A]
Hi,

I have a little problem connecting to the server. I noticed that when typing the IP address all characters change to lowercase, unfortunately, to connect to a my server some characters must be uppercase for exmaple: from pineapple to pineAPPle.

Is there any solution to be able to connect via such an address?

Best regards.

**Comment by faceincase** [comment]
Hello again,

IPv4 and IPv6 are not working. 
Some characters must be uppercase due to authorization from a server.
I realize that this is not the fault of the console - but the server, I would like to know if there is any chance to connect using uppercase characters in the subdomain.

Best regards.

**Comment by ORelio** [comment]
Created an Issue from your question (#1803) as this is currently not possible (we never encountered such a case until you reported it).
